main.go

```diff
@@ -557,6 +563,50 @@ func cmdEdit(c *cli.Context) error {
 	return cfg.runcmd(cfg.Editor, "", files...)
 }
 
+func cmdCat(c *cli.Context) error {
+	var cfg config
+	err := cfg.load()
+	if err != nil {
+		return err
+	}
+
+	var files []string
+	f, err := os.Open(cfg.MemoDir)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	files, err = f.Readdirnames(-1)
+	if err != nil {
+		return err
+	}
+	f.Close()
+	files = filterMarkdown(files)
+	var buf bytes.Buffer
+	err = cfg.runfilter(cfg.SelectCmd, strings.NewReader(strings.Join(files, "\n")), &buf)
+	if err != nil {
+		// TODO:
+		// Some select tools return non-zero, and some return zero.
+		// This part can't handle whether the command execute failure or
+		// the select command exit non-zero.
+		//return fmt.Errorf("%v: you need to install peco first: https://github.com/peco/peco", err)
+		return err
+	}
+	if buf.Len() == 0 {
+		return errors.New("No files selected")
+	}
+	files = strings.Split(strings.TrimSpace(buf.String()), "\n")
+
+	f, err = os.Open(filepath.Join(cfg.MemoDir, files[0]))
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	_, err = io.Copy(os.Stdout, f)
+	return err
+}
+
```

Comment:
This does not take second argument.

```
$ memo cat 2017-03-13-foo.md
```
This output content immediately.

Reply:
Yes I know, who is going to remember exact date and name, so I find that to be mute. But you are welcome to add it if you like.